vector/src/main/java/im/vector/activity/VectorRoomActivity.java

```diff
@@ -2214,7 +2224,8 @@ public void onClick(View v) {
 
                 if (unreadCount > 0) {
                     iconId = R.drawable.newmessages;
-                    textColor = R.color.vector_fuchsia_color;
+                    textColor = ThemeUtils.getColor(this, R.attr.vector_fuchsia_color);
```

Comment:
Some renaming could be done on attributes. Fuchsia is a color so its value shouldn't change depending on the theme. Ideally, the attribute should describe what it is, like "room background".
Many things are not very well named currently, that theme update could be an opportunity to rename some. :-)